boxes/make_boxes.sh

```diff
@@ -52,10 +54,10 @@ make_python3() {
     extract_deb "python3.5/libpython3.5-minimal_3.5.3-1_amd64.deb"
     extract_deb "expat/libexpat1_2.2.0-2+deb9u1_amd64.deb"
     extract_deb "zlib/zlib1g_1.2.8.dfsg-5_amd64.deb"
-    extract_deb "openssl/libssl1.1_1.1.0f-3+deb9u1_amd64.deb"
-    extract_deb "glibc/libc6_2.24-11+deb9u1_amd64.deb"
-    extract_deb "gcc-6/libgcc1_6.3.0-18_amd64.deb"
-    extract_deb "gcc-6/gcc-6-base_6.3.0-18_amd64.deb"
+    extract_deb "openssl/libssl1.1_1.1.0f-3+deb9u2_amd64.deb"
+    extract_deb "glibc/libc6_2.24-11+deb9u3_amd64.deb"
+    extract_deb "gcc-6/libgcc1_6.3.0-18+deb9u1_amd64.deb"
+    extract_deb "gcc-6/gcc-6-base_6.3.0-18+deb9u1_amd64.deb"
```

Comment:
Chyba trzeba będzie prędzej czy później przerobić to na coś bardziej maintainowalnego.
Mam gdzieś skrypt który używa debootstrapa i automatycznie znajduje zależności, jak chcesz to ci mogę podesłać.

Reply:
Yup, ale po kolei :)